source/features/small-user-avatars.css

```diff
@@ -3,3 +3,7 @@
 	margin-left: -3px;
 	margin-right: 2px !important;
 }
+
+.user-mention .rgh-small-user-avatars {
+	margin-left: 2px;
```

Comment:
Will `px` units scale correctly as the text size changes? `em` may be a better option.

Reply:
Avatar images are set to a fixed size by this feature, the margin would be correct regardless of text size